src/pages/App.tsx

```diff
@@ -14,44 +18,40 @@ import { useRouterPreference } from 'state/user/hooks'
 import { useIsDarkMode } from 'theme/components/ThemeToggle'
 import { flexRowNoWrap } from 'theme/styles'
 import { Z_INDEX } from 'theme/zIndex'
 import { STATSIG_DUMMY_KEY } from 'tracing'
 import { getEnvName, isBrowserRouterEnabled } from 'utils/env'
+import { getDownloadAppLink } from 'utils/openDownloadApp'
 import { getCurrentPageFromLocation } from 'utils/urlRoutes'
 import { getCLS, getFCP, getFID, getLCP, Metric } from 'web-vitals'
 
-import ErrorBoundary from '../components/ErrorBoundary'
-import NavBar, { PageTabs } from '../components/NavBar'
-import Polling from '../components/Polling'
-import Popups from '../components/Popups'
-import DarkModeQueryParamReader from '../theme/components/DarkModeQueryParamReader'
-import { getDownloadAppLink } from '../utils/openDownloadApp'
-import AddLiquidity from './AddLiquidity'
-import { RedirectDuplicateTokenIds } from './AddLiquidity/redirects'
-import { RedirectDuplicateTokenIdsV2 } from './AddLiquidityV2/redirects'
+// High-traffic pages (index and /swap) should not be lazy-loaded.
 import Landing from './Landing'
-import MigrateV2 from './MigrateV2'
-import MigrateV2Pair from './MigrateV2/MigrateV2Pair'
-import NotFound from './NotFound'
-import Pool from './Pool'
-import PositionPage from './Pool/PositionPage'
-import PoolV2 from './Pool/v2'
-import PoolFinder from './PoolFinder'
-import RemoveLiquidity from './RemoveLiquidity'
-import RemoveLiquidityV3 from './RemoveLiquidity/V3'
 import Swap from './Swap'
-import { RedirectPathToSwapOnly } from './Swap/redirects'
-import Tokens from './Tokens'
 
-const TokenDetails = lazy(() => import('./TokenDetails'))
-const PoolDetails = lazy(() => import('./PoolDetails'))
-const Vote = lazy(() => import('./Vote'))
 const NftExplore = lazy(() => import('nft/pages/explore'))
 const Collection = lazy(() => import('nft/pages/collection'))
 const Profile = lazy(() => import('nft/pages/profile/profile'))
 const Asset = lazy(() => import('nft/pages/asset/Asset'))
+const AddLiquidity = lazy(() => import('pages/AddLiquidity'))
+const RedirectDuplicateTokenIds = lazy(() => import('pages/AddLiquidity/redirects'))
+const RedirectDuplicateTokenIdsV2 = lazy(() => import('pages/AddLiquidityV2/redirects'))
+const MigrateV2 = lazy(() => import('pages/MigrateV2'))
+const MigrateV2Pair = lazy(() => import('pages/MigrateV2/MigrateV2Pair'))
+const NotFound = lazy(() => import('pages/NotFound'))
+const Pool = lazy(() => import('pages/Pool'))
+const PositionPage = lazy(() => import('pages/Pool/PositionPage'))
+const PoolV2 = lazy(() => import('pages/Pool/v2'))
+const PoolDetails = lazy(() => import('pages/PoolDetails'))
+const PoolFinder = lazy(() => import('pages/PoolFinder'))
+const RemoveLiquidity = lazy(() => import('pages/RemoveLiquidity'))
+const RemoveLiquidityV3 = lazy(() => import('pages/RemoveLiquidity/V3'))
+const TokenDetails = lazy(() => import('pages/TokenDetails'))
+const Tokens = lazy(() => import('pages/Tokens'))
```

Comment:
are these low-traffic enough to make lazy?

Reply:
i'm not making a very informed decision, just making the "normal" landing pages eager and the rest lazy.